src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/58aa8d2b83.js" crossorigin="anonymous"></script>
+    
+    <title>ToDoList</title>
+</head>
+<body>
+    <div class="todolist_container">
+      <div class="todolist_header"><h5>My Todolist</h5>
```

Comment:
- [ ] You can use semantic HTML5 instead, for example \<section\> tag

Reply:
Hi @Ismailco 
thanks for the time you spent reviewing my work, and for the valuable comments, yes I need to use semantic tags 👍